department-of-veterans-affairs/caseflow
# Div/appeals 35473

Resolves [Intake Form: Ability to resume from Correspondence Queue(https://jira.devops.va.gov/browse/APPEALS-35473)

# Description
Provided the ability to navigate to the intake form, and from there we can cancel and return to Step 3

## Acceptance Criteria


## Testing Plan
1. Login with [JOLLY_POSTMAN (VACO)]
2. Navigate to Your correspondence in-progress tab
3. Start the intake process.
4. Verify that you are able to navigate through the steps 1, 2, 3.
5. At any step, initiate the cancellation process.
6. Confirm that the cancellation process works as expected and redirects you to the previous page.
7. Verify that you are able to resume the intake process from the step 3


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
